common/src/main/java/com/linecorp/centraldogma/internal/Jackson.java

```diff
@@ -281,6 +295,66 @@ public static String escapeText(String text) {
         return new String(enc.quoteAsString(text));
     }
 
+    public static JsonNode mergeJsonNodes(JsonNode... jsonNodes) {
+        return mergeJsonNodes(ImmutableList.copyOf(requireNonNull(jsonNodes, "jsonNodes")));
+    }
+
+    public static JsonNode mergeJsonNodes(Iterable<JsonNode> jsonNodes) {
```

Comment:
How about `mergeTree` for consistency with `extractTree`?

Reply:
Renamed to `mergeTree`. Thank you~!